producer.robot

```diff
@@ -11,7 +11,7 @@ Produce items
     ...    Get source Excel file from work item.
     ...    Read rows from Excel.
     ...    Creates output work items per row.
-    ${path}=    Get Work Item File    orders.xlsx    %{ROBOT_ARTIFACTS}${/}orders.xlsx
+    ${path}=    Get Work Item File    orders.xlsx    ${OUTPUT_DIR}${/}orders.xlsx
```

Comment:
What is the reason for this change ?

Reply:
These need to be `ROBOT_ARTIFACTS` the stuff that is pushed to Control Room is separated by this and needs to be clear.. robot outputs can go in another directory. 
E.g. customer does not want rfw log.html to Cloud but needs a few files to go there.

Reply:
Good to know, there was a strange change before this diff from `%{ROBOT_ARTIFACTS}` -> `%{ROBOT_ARTIFACT}` and from there this change stemmed.

Reply:
I have never used nor seen this variable so that was why I changed it because I expected my test to fail because this environment variable isn't defined and I didn't see it in any of the documentation. Are there other such run time variables that we need to define in our documentation that is not included in the Robot Framework user guide?